packages/adapter-d1/src/index.ts

```diff
@@ -0,0 +1 @@
+export const TEMP = 42
```

Comment:
```suggestion
export const ANSWER = 42
```

Reply:
:trollface: Obviously, we also need a comment mentioning https://en.wikipedia.org/wiki/42_(number)